src/test/java/core/basesyntax/service/impl/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class CsvFileReaderImplTest {
+
+    private static List<String> expected = new ArrayList<>();
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @BeforeClass
+    public static void setUp() {
+        expected.add("type,fruit,quantity");
+        expected.add("b,banana,20");
+        expected.add("b,apple,100");
+        expected.add("s,banana,100");
+        expected.add("p,banana,13");
+        expected.add("r,apple,10");
+        expected.add("p,apple,20");
+        expected.add("p,banana,5");
+        expected.add("s,banana,50");
+    }
+
+    @Test
+    public void readFile_Ok() {
+        String filePath = "src/main/java/core/basesyntax/resources/Input.csv";
+        List<String> actual = new CsvFileReaderImpl().readFromFile(filePath);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void readFile_notOk() {
```

Comment:
```suggestion
    public void readFromFile_invalidPath_NotOk() {
```

fix method names in all tests